staging/src/k8s.io/apimachinery/Godeps/Godeps.json

```diff
@@ -217,6 +369,14 @@
 		{
 			"ImportPath": "k8s.io/kube-openapi/pkg/common",
 			"Rev": "39a7bf85c140f972372c2a0d1ee40adbf0c8bfe1"
+		},
+		{
+			"ImportPath": "k8s.io/kube-openapi/pkg/util/proto",
```

Comment:
@deads2k @lavalamp any objection to letting k8s.io/apimachinery depends on k8s.io/kube-openapi?